dashboard/app/views/admin_reports/retention.html.haml

```diff
@@ -7,6 +7,13 @@
 
 %br
 
+%form{method: 'get'}
+  Script IDs:
+  %input{name: "script_ids", id: "script_ids", value: params[:script_ids]}
```

Comment:
could you add something explaining that this is expected to be a comma-separated list of ids? Perhaps a placeholder?